labels_command.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+
+// ! IMPORTANT !
+// - Any labels on the GitHub repo that are not defined in LABELS_COLORS will be deleted
+// - Any labels in LABELS_COLORS that do not exist on the GitHub repo will be created
+//
+// Do not prefix color with hash i.e. use 'cccccc' not '#cccccc'
+const LABELS_COLORS = [
+    'affects/v4' => '5319e7',
+    'affects/v5' => '0e8a16',
+    'complexity/low' => 'c2e0c6',
+    'complexity/medium' => 'fef2c0',
+    'complexity/high' => 'f9d0c4',
+    'Epic' => '3e4b9e',
+    'impact/low' => 'fef2c0',
+    'impact/medium' => 'f7c6c7',
+    'impact/high' => 'eb6420',
+    'impact/critical' => 'e11d21',
+    'rfc/accepted' => 'dddddd',
+    'rfc/draft' => 'dddddd',
+    'type/api-break' => '1d76db',
+    'type/bug' => 'd93f0b',
+    'type/docs' => '02d7e1',
+    'type/enhancement' => '0e8a16',
+    'type/userhelp' => 'c5def5',
+    'type/UX' => '006b75',
+    'type/other' => '975515',
+];
+
+// Rename existing labels 'from' => 'to'
+const LABELS_RENAME = [
+    'effort/easy' => 'complexity/low',
+    'effort/medium' => 'complexity/medium',
+    'effort/hard' => 'complexity/high',
+    'change/major' => 'type/api-break',
+    'type/api-change' => 'type/api-break',
+];
+
+$labelsCommand = function(InputInterface $input, OutputInterface $output): int {
+    // This is the code that is executed when running the 'labels' command
+
+    // variables
+    global $OUT, $REPOS_WITH_LABELS_UPDATED;
+    $OUT = $output;
+
+    // validate system is ready
+    validate_system();
+
+    // setup directories
+    setup_directories($input);
+
+    // CMS major version to use
+    $cmsMajor = $input->getOption('cms-major') ?: CURRENT_CMS_MAJOR;
+
+    // modules
+    $modules = filtered_modules($cmsMajor, $input);
```

Comment:
The labels command should just do all of the modules - if it's a supported module, it should have the correct labels.

Reply:
filtered_modules() will remove supported modules based any optional --only --exclude command line options

Reply:
I don't mind `filtered_modules()`, but we shouldn't be limiting to a specific CMS major. All supported modules should be updated, regardless of which major release they're supported for.

Reply:
Have to work around this because we don't have an "endpoint" that says "these are the currently supported CMS versions"

Because of this, we cannot automatically work out if we should use one or both of these
- https://raw.githubusercontent.com/silverstripe/supported-modules/4/modules.json
- https://raw.githubusercontent.com/silverstripe/supported-modules/5/modules.json

I've updated the README.md to say you need to run this command for every currently supported cms-major

Reply:
`CURRENT_CMS_MAJOR` says what the current heighest supported major is - so just have it do `CURRENT_CMS_MAJOR` and `CURRENT_CMS_MAJOR - 1` (which is 5 and 4 right now, and will be 6 and 5 when 6 is tagged stable)

Reply:
Have updated to use modules from (current cms major) + ((current cms major) - 1)